extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -41,7 +40,10 @@ export class OpenPGPKey {
   };
 
   public static validateAllDecryptedPackets = async (key: OpenPGP.key.Key): Promise<void> => {
-    for (const prvPacket of key.toPacketlist().filter(OpenPGPKey.isPacketPrivate).filter(packet => packet.isDecrypted())) {
+    for (const prvPacket of key
+      .toPacketlist()
+      .filter(OpenPGPKey.isPacketPrivate)
+      .filter(packet => packet.isDecrypted())) {
```

Comment:
I'm thinking maybe the `) {` could be on its own line, it would be a bit more readable

Reply:
Updated it to
```js
const decryptedPackets = key
  .toPacketlist()
  .filter(OpenPGPKey.isPacketPrivate)
  .filter(packet => packet.isDecrypted());
for (const prvPacket of decryptedPackets) {
  await prvPacket.validate(); // gnu-dummy never raises an exception, invalid keys raise exceptions
}
```